GitUI/UserControls/RevisionGrid/Columns/BuildStatusColumnProvider.cs

```diff
@@ -185,7 +185,7 @@ public override bool TryGetToolTip(DataGridViewCellMouseEventArgs e, GitRevision
             if (revision.BuildStatus is not null)
             {
                 toolTip = revision.BuildStatus.Tooltip ?? revision.BuildStatus.Description;
-                return true;
+                return toolTip is not null;
```

Comment:
`Description` can be null, so it was possible before to return null and true, which is not expected by callers.